src/core/style/mixins.scss

```diff
@@ -125,15 +125,10 @@
   }
 }
 
-@mixin rtl-prop($ltr-prop, $rtl-prop, $value) {
+@mixin rtl-prop($ltr-prop, $rtl-prop, $value, $reset-value) {
```

Comment:
I would say you should add a `null` default: 
`@mixin rtl-prop($ltr-prop, $rtl-prop, $value, $reset-value: null) {`

That was, we can make some segments with even less CSS. 

http://www.sassmeister.com/gist/b87f3d64603d5bceaa29534ffc585457